### ISSUE TITLE
Failed to load resource: the server responded with a status of 403 (OK)

### ISSUE THREAD

**Issue by fbukevin** [googleplus/gplus-quickstart-javascript#14]
I downloaded this example and changed all YOUR_CLIENT_ID with mine.

After I launched server and connect to my set url http://localhost:3456

I still failed to access my profile as following two screenshot.

<img width="1280" alt="2015-11-27 12 06 43" src="https://cloud.githubusercontent.com/assets/4548874/11433971/195e1862-9500-11e5-83fc-6701f524b978.png">

<img width="1275" alt="2015-11-27 12 06 58" src="https://cloud.githubusercontent.com/assets/4548874/11433979/23c1952c-9500-11e5-93b4-13d605cf67e8.png">

**Comment by gguuss** [comment]
I just tested this on Chrome for OS X and didn't have any issues. Did you add `http://localhost:3456` to your authorized JavaScript origins when setting up your OAuth 2.0 credentials?

**Comment by gguuss** [comment]
One more thing, have you enabled the Google+ API?

**Comment by frank-carnovale** [comment]
I also had that problem but the fix was indeed to enable the Google+ API.

**Comment by gguuss** [comment]
@fbukevin - were you able to resolve the problem?  If so, please close the issue.

**Comment by fbukevin** [comment]
Sorry for missing reply. I fixed this problem, too. I also resolved it with enable the Google+ API.
I only added a new credential in developer console originally.

Here is what the setup page looks like.
![1](https://cloud.githubusercontent.com/assets/4548874/12437839/614e944a-bf5c-11e5-8312-bb5d43b24fab.png)

And enable here.
![2](https://cloud.githubusercontent.com/assets/4548874/12437858/84d86940-bf5c-11e5-8d4f-87633b2346c9.png)
